packages/samples/headless-commerce-ssr-remix/app/components/standalone-search-box.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import {
+  useInstantProducts,
+  useRecentQueriesList,
+  useStandaloneSearchBox,
+} from '@/lib/commerce-engine';
+import {useNavigate} from '@remix-run/react';
+import {useEffect, useState} from 'react';
+import InstantProducts from './instant-product';
+import RecentQueries from './recent-queries';
+
+export default function StandaloneSearchBox() {
```

Comment:
The query is not getting executed when redirecting to the search page.

Reply:
I'm surprised that using the fragment works at all; it's not supposed to be passed down to the server...

Reply:
It isnt, it only works client-side

Reply:
ah I guess it's a remix helper that's why

regarding the clientId, I'll have to investigate; it's definitely set cause when I print it out in the cart's loader I see it in the server logs